### ISSUE TITLE
CiscoShipped has shut down, have you considered porting this to AWS Lambda or ECS?

### ISSUE THREAD

**Issue by msteelepmp** [robertcsapo/cisco-meraki-cmx-api-app#2]

**Comment by robertcsapo** [comment]
@msteelepmp there's a Cisco Meraki CMX repo. for Lambda here [https://github.com/dexterlabora/cmxreceiver-lambda-inline](https://github.com/dexterlabora/cmxreceiver-lambda-inline)

I haven't looked in to AWS ECS. Does it have CI/DI integration that you're looking for?

**Comment by msteelepmp** [comment]
Thanks, that’s great!  

I got your Docker image working in AWS ECS just fine.  It was a manual process using the AWS ECS Wizard. I just had to specify the port and Environment vars in the config.

Launching your Docker image in AWS ECS is certainly scriptable via the AWS and Docker CLI. But I’ve done very little of this so cannot comment further. Half of what I did was CLI. If you’re interested in automating a AWS ECS version, I can send you the steps I used, but it won’t be pretty :)

What you’ve put together is great, but I think the Lambda solution will better meet my needs.

Thanks again!
